provider/resources.go

```diff
@@ -92,24 +92,787 @@ func Provider() tfbridge.ProviderInfo {
 			// },
 		},
 		PreConfigureCallback: preConfigureCallback,
-		Resources:            map[string]*tfbridge.ResourceInfo{
-			// Map each resource in the Terraform provider to a Pulumi type. Two examples
-			// are below - the single line form is the common case. The multi-line form is
-			// needed only if you wish to override types or other default options.
-			//
-			// "aws_iam_role": {Tok: tfbridge.MakeResource(mainPkg, mainMod, "IamRole")}
-			//
-			// "aws_acm_certificate": {
-			// 	Tok: tfbridge.MakeResource(mainPkg, mainMod, "Certificate"),
-			// 	Fields: map[string]*tfbridge.SchemaInfo{
-			// 		"tags": {Type: tfbridge.MakeType(mainPkg, "Tags")},
-			// 	},
-			// },
+		Resources: map[string]*tfbridge.ResourceInfo{
+			"ibm_api_gateway_endpoint":                               {Tok: tfbridge.MakeResource(mainPkg, mainMod, "ApiGatewayEndpoint")},
+			"ibm_api_gateway_endpoint_subscription":                  {Tok: tfbridge.MakeResource(mainPkg, mainMod, "ApiGatewayEndpointSubscription")},
+			"ibm_app":                                                {Tok: tfbridge.MakeResource(mainPkg, mainMod, "App")},
+			"ibm_app_config_environment":                             {Tok: tfbridge.MakeResource(mainPkg, mainMod, "AppConfigEnvironment")},
```

Comment:
@rshade I wouldn't put everything at the root level (aka `mainMod`, which is `"index"`). For everything currently prefixed with `App`, I would use the module name `app`:

```
"ibm_app_config_environment": tfbridge.MakeResource(mainPkg, "app", "ConfigEnvironment")}, 
"ibm_app_config_feature": tfbridge.MakeResource(mainPkg, "app", "ConfigFeature")},

Reply:
Should App Be lower or uppercase?

Reply:
Looking at the different module names of the `aws` provider, the values are camel-cased:

https://github.com/pulumi/pulumi-aws/blob/master/provider/resources.go#L39-L207